README.md

```diff
@@ -16,20 +16,19 @@ If you're a Windows user, please consider using [WSL](https://learn.microsoft.co
 
 ### Extra dependencies
 
-Graph compilation requires non-Python dependencies to be installed.
-Either:
-1. Install a recent Julia version from the [Julia website](https://julialang.org/downloads/).
-2. Make sure `julia` executable is on your `$PATH`. You can test it by running `julia` in a new terminal window.
-3. Install Julia dependencies: open `julia` REPL, press `]`, run `add Jabalizer`.
+#### Julia
+If you want to run the Graph State Compilation pipeline, you'll have to install Julia. Fortunately, benchq will install Julia for you whenever it is needed.
```

Comment:
Is it correct that benchq will only install julia if you do not already have the correct version already installed? Might be good to clarify this.